tests/pass/concurrency/linux-futex.rs

```diff
@@ -137,23 +136,29 @@ fn wait_wake() {
     let t = thread::spawn(move || {
         thread::sleep(Duration::from_millis(200));
         unsafe {
-            assert_eq!(libc::syscall(
-                libc::SYS_futex,
-                &FUTEX as *const i32,
-                libc::FUTEX_WAKE,
-                10, // Wake up at most 10 threads.
-            ), 1); // Woken up one thread.
+            assert_eq!(
+                libc::syscall(
+                    libc::SYS_futex,
+                    &FUTEX as *const i32,
+                    libc::FUTEX_WAKE,
+                    10, // Wake up at most 10 threads.
+                ),
+                1, // Woken up one thread.
```

Comment:
It was clear to me that this comment refers to the `1`, not the assert as a whole.